setup.py

```diff
@@ -30,10 +30,5 @@ def read(fname):
     ],
     packages=["ansible_deployer"],
     include_package_data=True,
-    install_requires=["pyyaml>=5.3.1", "cerberus>=1.3.4"],
-    entry_points={
```

Comment:
pip install without entry_points sounds incorreect if it's supposed to provide application

Reply:
Looks like setup.py is required by pylint, otherwise I would remove it completely.